### PR TITLE
Makes the autopilot and current-list-of-speakers ready for OS4

### LINKED ISSUE #328: List of speakers is missing in the cinema/AP
The whole logic to `CLOS` does not work currently.

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
- The <cinema />-component and the <list-of-speakers />-component requests the CLOS of the active-meeting
- Creates a constant "Follow"-object to request the CLOS of a meeting

Fixes #328.

### CODE REVIEW

File: client/src/app/site/projector/services/current-list-of-speakers.service.ts [hunk 1]
```diff
@@ -2,13 +2,39 @@ import { Injectable } from '@angular/core';
...
+                                { idField: 'speaker_ids', follow: [{ idField: 'user_id', fieldset: 'shortName' }] }
+                            ]
+                        }
+                    ]
+                }
+            ],
+            fieldset: 'content'
+        }
+    ]
+};
```
Comment (GabrielInTheWorld) [inline client/src/app/site/projector/services/current-list-of-speakers.service.ts:36]: I don't know, if we achieve this in a smarter way...

Comment (FinnStutzenstein) [reply client/src/app/site/projector/services/current-list-of-speakers.service.ts:36]: No, this is right - you request it exactly as the requirements say: "the autopilot shows the username of the list of speakers of the currently projected slide on the reference projector". It just resembles this perfectly.

File: client/src/app/site/cinema/components/cinema/cinema.component.ts [hunk 2]
```diff
@@ -120,4 +126,13 @@ export class CinemaComponent extends BaseComponent implements OnInit {
     public async readdLastSpeaker(): Promise<void> {
         await this.listOfSpeakersRepo.readdLastSpeaker(this.listOfSpeakers).catch(this.raiseError);
     }
+
+    private init(): void {
+        this.requestModels({
```
Comment (FinnStutzenstein) [inline client/src/app/site/cinema/components/cinema/cinema.component.ts:131]: The BaseModelContextComponent has a `getModelRequest`. Just use this.

File: client/src/app/site/cinema/components/cinema/cinema.component.ts [hunk 3]
```diff
@@ -106,7 +110,9 @@ export class CinemaComponent extends BaseComponent implements OnInit {
     }
 
     public ngOnInit(): void {
+        super.ngOnInit();
         super.setTitle('Autopilot');
+        this.init();
```
Comment (FinnStutzenstein) [inline client/src/app/site/cinema/components/cinema/cinema.component.ts:115]: No need for the init method.

File: client/src/app/site/projector/services/current-list-of-speakers.service.ts [hunk 4]
```diff
@@ -130,21 +153,11 @@ export class CurrentListOfSpeakersService {
...
-            try {
-                const viewModel = this.projectorService.getViewModelFromIdentifiableProjectorElement(
-                    identifiableNonStableElement
-                );
-                if (isBaseViewModelWithListOfSpeakers(viewModel)) {
-                    return viewModel.listOfSpeakers;
-                }
-            } catch (e) {
-                // make TypeScript silent.
+        for (const projection of projector.current_projections) {
```
Comment (FinnStutzenstein) [inline client/src/app/site/projector/services/current-list-of-speakers.service.ts:156]: Nice, having shorter code as in OS3 is an indication, that we did something right with the new concept ;)

File: client/src/app/site/agenda/components/list-of-speakers/list-of-speakers.component.ts [hunk 5]
```diff
@@ -149,6 +155,15 @@ export class ListOfSpeakersComponent extends BaseModelContextComponent implement
         });
     }
 
+    private requestCurrentListOfSpeakers(): void {
```
Comment (FinnStutzenstein) [inline client/src/app/site/agenda/components/list-of-speakers/list-of-speakers.component.ts:158]: If this is used once, move it up.

**Review by FinnStutzenstein — CHANGES_REQUESTED** [review]
Minor remarks, not tested

**Review by FinnStutzenstein — APPROVED** [review]

**Pull request merged**